index.html

```diff
@@ -157,35 +157,25 @@ <h3>Full Name</h3>
             <div class="custom-card-wrapper col-12 col-md-6 col-lg-4">
               <div class="custom-card">
                 <div class="avatar-wrapper">
-                  <img src="img/placeholder.jpeg" alt="" class="avatar" />
+                  <img src="img/aog.jpg" alt="" class="avatar" />
                 </div>
                 <div>
-                  <h3>Full Name</h3>
+                  <h3>Ghita Alexandru Ovidiu</h3>
                   <p>
-                    Lorem ipsum dolor sit amet consectetur, adipisicing elit.
-                    Molestiae omnis nobis quo dicta culpa eum exercitationem
-                    provident ex iure pariatur? Tempora sunt accusamus tempore
-                    modi.
+                    Hey, im an ex-medical engineer witch has gone through IT school web development program, in order to change my carrer path.
```

Comment:
`engineer witch has gone` typo here, should be: `engineer which has gone`

Reply:
`IT school web development program` should be `IT School's web development program`

Reply:
`in order to change my carrer path` typo here, should be `in order to change my career path`

Reply:
I think i did it